### PR TITLE
[Ft] Nodes last notify

### PR DESCRIPTION
**Pull request by KhadijaMahanga**
## Description
- Saves node's `last-notify` when posting sensors data
- Removes `last-data-push`  on Node

Screenshot from 2021-01-29 17-14-26

Fixes # (issue)

## Type of change

- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation

### CODE REVIEW

**Review by esirK — APPROVED** [review]

File: sensorsafrica/api/v1/serializers.py [hunk 1]
```diff
@@ -25,3 +38,15 @@ class SensorDataSerializer(serializers.ModelSerializer):
     class Meta:
         model = SensorData
         fields = ['location', 'timestamp', 'sensordatavalues']
+
+class VerboseSensorDataSerializer(PostSensorDataSerializer):
```
Comment (kilemensi) [inline sensorsafrica/api/v1/serializers.py:42]:
```suggestion
class LastNotifySensorDataSerializer(PostSensorDataSerializer):
```

**Review by kilemensi — APPROVED** [review]
🚀 @KhadijaMahanga 

Any performance issues (slow down, etc.) when posting data?

**Comment by KhadijaMahanga** [comment]
>  @KhadijaMahanga
> 
> Any performance issues (slow down, etc.) when posting data?

@kilemensi yes, I have observed a difference of 0.008sec on average.

**Comment by kilemensi** [comment]
> > @KhadijaMahanga
> > Any performance issues (slow down, etc.) when posting data?
> 
> @kilemensi yes, I have observed a difference of 0.008sec on average.

That doesn't sound too bad @KhadijaMahanga

**Pull request merged**